packages/camera/tizen/src/camera_plugin.cc

```diff
@@ -241,9 +246,22 @@ class CameraPlugin : public flutter::Plugin {
       }
       result->Error("InvalidArguments", "Please check 'zoom'");
     } else if (method_name == "lockCaptureOrientation") {
-      result->NotImplemented();
+      if (method_call.arguments()) {
+        flutter::EncodableMap arguments =
+            std::get<flutter::EncodableMap>(*method_call.arguments());
+        std::string orientation;
+        OrientationType orientation_type;
+        if (GetValueFromEncodableMap(arguments, "orientation", orientation) &&
+            StringToOrientationType(orientation, orientation_type)) {
+          camera_->LockCaptureOrientation(orientation_type);
+          result->Success();
+          return;
+        }
+        result->Error("InvalidArguments", "Please check 'orientation'");
+      }
```

Comment:
How about putting some logs here?  I mean for error case handling

Reply:
It's just a suggestion... I guess you know the reason :imp:

Reply:
Well... I want to reduce unnecessary logs as much as possible. in this case, throwing an error directly to ```flutter``` and leaving enough logs in the wrapper implementation. so I think it is enough

Reply:
Got it.